### ISSUE TITLE
Markdown Autofocus with modal

### ISSUE THREAD

**Issue by Necro-master** [Megabit/Blazorise#3713]
**Describe the bug**
Markdown Autofocus inside modal doesn't work

**To Reproduce**

***Markdown Edit Modal Component***

```
<Tooltip Text="@("Edit " + PropertyName)">
    <Button class="btn mb-1" Clicked="@ModalShow" too>
        <i class="fa fa-pencil"></i>
    </Button>
</Tooltip>
<Modal @ref="modal">
    <Form>
        <ModalContent Centered>
            <ModalHeader>
                <ModalTitle>Edit @PropertyName</ModalTitle>
            </ModalHeader>
            <ModalBody>
                <Field>
                    <Markdown Autofocus="true"/>
                </Field>
            </ModalBody>
            <ModalFooter>
            </ModalFooter>
        </ModalContent>
    </Form>
</Modal>
```

**Expected behavior**
Autofocused Markdown

**Additional context**
EasyMDE Autofocus bug was already fixed(in easymde@2.16.2-49.0).

**Comment by stsrki** [comment]
Moving to 1.1 since the fix would require adding an additional API.

**Comment by stsrki** [comment]
Fixed for 1.1
